programs/ziti-edge-tunnel/package/systemd/ziti-edge-tunnel.env.in

```diff
@@ -1,3 +1,9 @@
+# all enrollment tokens named *.jwt are consumed and replaced with identity JSON files to be loaded at startup
 ZITI_IDENTITY_DIR='@ZITI_IDENTITY_DIR@'
+
+# reserved dynamic IP range for proxied services
 ZITI_DNS_IP_RANGE='100.64.0.1/10'
+
+# the log level specified in @ZITI_STATE_DIR@/config.json has higher precedence than this env var; delete or modify that
```

Comment:
explain the extremely confusing precedence of the ZET log level ([link to issue](https://github.com/openziti/ziti-tunnel-sdk-c/issues/531))